extensions/gitlab-docs/src/pajamas.tsx

```diff
@@ -0,0 +1,161 @@
+import { ActionPanel, Action, List, showToast, Toast } from "@raycast/api";
+import { useState, useEffect, useRef, useCallback } from "react";
+import fetch, { AbortError } from "node-fetch";
+import { LocalStorage } from "@raycast/api";
+
+const baseUrl = "https://design.gitlab.com/";
+const apiUrl = `${baseUrl}/_nuxt/search-index/en.json`;
+
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List
+      isLoading={state.isLoading}
+      onSearchTextChange={search}
+      searchBarPlaceholder="Search GitLab Design System..."
+      throttle
+    >
+      <List.Section title="Results" subtitle={state.metas.length + ""}>
+        {state.metas.map((searchResult) => (
+          <SearchListItem key={searchResult.key} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      icon="../assets/pajamas-icon.png"
+      title={searchResult.name}
+      accessoryTitle={searchResult.category}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <Action.OpenInBrowser title="Show Details" url={searchResult.url} />
```

Comment:
This title is usually used when opening the Detail Panel; maybe use the default title?